src/NosCore.GameObject/Services/MapInstanceAccess/MapInstance.cs

```diff
@@ -153,9 +153,9 @@ public MapItem PutItem(short amount, IItemInstance inv, ClientSession session)
             MapItem droppedItem = null;
             List<MapCell> possibilities = new List<MapCell>();
 
-            for (short x = -2; x < 3; x++)
+            for (short x = -1; x < 2; x++)
```

Comment:
why ?

Reply:
Dropping items more far than in official

Reply:
are you sure ? i kinda remember it was +2 -2 but maybe it's only +1-1

Reply:
Actual NosCore -> https://gyazo.com/70b876b091b034e363179375d59e41de
Modified NosCore -> https://gyazo.com/4f1b969faf895d509b59b84c27b3d26a